lib/service/Service.ts

```diff
@@ -48,13 +51,54 @@ class Service extends EventEmitter {
     this.shutdown = components.shutdown;
 
     this.orderBook = components.orderBook;
-    this.lndClient = components.lndClient;
+    this.lndBtcClient = components.lndBtcClient;
+    this.lndLtcClient = components.lndLtcClient;
     this.raidenClient = components.raidenClient;
     this.pool = components.pool;
     this.config = components.config;
 
     this.logger = Logger.rpc;
   }
+  private getLndInfo = async (config: LndClientConfig, lndClient: LndClient)  => {
+
+    let info: any = {};
+
+    if (config.disable) {
```

Comment:
I think it's better to call lndClient.isDisabled() here to determine whether lnd is disabled or simply disconnected. Previously we were using the config because the lnd client status didn't make a distinction between being disabled or merely disconnected, but I think it's good to have that distinction and we should take advantage of it here.

Reply:
Done